### PR TITLE
Update field names to be consistent with FLEx

### LINKED ISSUE #1730: update labels to be more consistent with FLEx
This card is about changing the labels the user sees for the "word" and "meaning" fields.

Word -> Lexeme Form (or "Lexeme" - need clarification)
Meaning -> Sense

### PR DESCRIPTION
**Pull request by longrunningprocess**
## Description

| LF labels | FLEx labels |
| --------- | ----------- |
| Word       | Lexeme Form |
| Meaning | Sense              |
| Part of Speech | Grammatical Category |
| Type | Sense Type |

## Checklist

- [x] I have labeled my PR with: bug, feature, engineering, security fix or testing
- [x] I have performed a self-review of my own code
- [x] I have reviewed the title & description of this PR which I will use as the squashed PR commit message
- [x] I have updated tests that prove my fix is effective or that my feature works
- [x] I have enabled auto-merge (optional)

## Testing

Testers, use the following instructions against our staging environment. Post your findings as a comment and include any meaningful screenshots, etc.

- Ensure field names referenced in table above have changed to the correct label
- Ensure there are no more old field name references in the project configuration area

### CODE REVIEW

**Comment by longrunningprocess** [comment]
moved to draft because we received more info from the requestor on fields, we'll need to do a little more work once final decisions are made...

File: src/Api/Model/Languageforge/Lexicon/Config/LexConfig.php [hunk 1]
```diff
@@ -127,14 +127,14 @@ public static function flexOptionlistCode($fieldName)
...
-        "grammatical-info" => "Part of Speech",
+        "grammatical-info" => "Grammatical Category",
         "semantic-domain-ddp4" => "Semantic Domain",
         "domain-type" => "Academic Domains",
         self::ENVIRONMENTS => "Environments",
         "location" => "Location",
         "usage-type" => "Usages",
         "reversal-type" => "Reversal Entries",
-        "sense-type" => "Type",
+        "sense-type" => "Sense Type",
```
Comment (megahirt) [inline src/Api/Model/Languageforge/Lexicon/Config/LexConfig.php:137]: I would like you to double check that the label in FLEx is actually "Sense Type".  It already falls under the "Sense" heading, so to me the label "Sense Type" is redundant.  However, if Verna could not easily determine what the field was, maybe that's an argument for "Sense Type".

I am not on Windows, so cannot easily check.  I knew that eventually you will need to be able to fire up FLEx (on Windows presumably) so now's a good time to figure out how to do that :). FLEx does run on Linux, although the setup is harder (it's more than a file download like a windows installer is)

**Review by megahirt — APPROVED** [review]
Other than verifying what "Sense Type" field label is used in FLEx, I approve of this PR.  Thanks!  This PR should match what FLEx has as far as field labels go.

File: src/Api/Model/Languageforge/Lexicon/Config/LexConfig.php (hunk 1 above)
Comment (myieye) [reply src/Api/Model/Languageforge/Lexicon/Config/LexConfig.php:137]: "Sense Type" is a Sense field in FLEx
image

Comment (megahirt) [reply src/Api/Model/Languageforge/Lexicon/Config/LexConfig.php:137]: @myieye to the rescue...now @longrunningprocess can put off installing Windows for another day 😆

Comment (longrunningprocess) [reply src/Api/Model/Languageforge/Lexicon/Config/LexConfig.php:137]: thank you more than words can convey @myieye  😆

Comment (myieye) [reply src/Api/Model/Languageforge/Lexicon/Config/LexConfig.php:137]: @megahirt I actually somehow didn't notice your second paragraph about @longrunningprocess installing FLEx 😆. W00ps 😁.

**Pull request merged**